lib/constants/category.ts

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line typescript-enum/no-enum
```

Comment:
Why do you prefer enum for this?

Reply:
I wanted to lean on the current implementation language as much as possible to ease the review, but there is no technical reason to use enums.

Reply:
i like to avoid enum's